tests/lib/rules/forbid-prop-types.js

```diff
@@ -709,20 +722,6 @@ ruleTester.run('forbid-prop-types', rule, {
       '});'
     ].join('\n'),
     errors: 2
-  }, {
-    code: [
-      'var First = createReactClass({',
-      '  propTypes: {',
-      '    s: PropTypes.shape({,',
```

Comment:
This test code was throwing parse error :(

Reply:
Why? what parse error?

Reply:
Oops, That was my misunderstanding.

Reply:
The change must revert.. 😓

Reply:
Oops, Now, I checked with ESLint v7.

The test is throwing error...

```
  1) forbid-prop-types
       invalid
         var First = createReactClass({
  propTypes: {
    s: PropTypes.shape({,
      o: PropTypes.object
    })
  },
  render: function() {
    return <div />;
  }
});:

      AssertionError [ERR_ASSERTION]: A fatal parsing error occurred: Parsing error: Unexpected token ,
      + expected - actual

      -false
      +true

      at runRuleForItem (node_modules/eslint/lib/rule-tester/rule-tester.js:559:13)
      at testInvalidTemplate (node_modules/eslint/lib/rule-tester/rule-tester.js:650:28)
      at Context.RuleTester.it (node_modules/eslint/lib/rule-tester/rule-tester.js:874:25)
```

Reply:
`Comma` may have been the cause ...??

```
{,
  o: PropTypes.object
}
```